hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/CommandQueue.java

```diff
@@ -38,9 +40,8 @@
  * there where queued.
  */
 public class CommandQueue {
-  // This list is used as default return value.
-  private static final List<SCMCommand> DEFAULT_LIST = new ArrayList<>();
   private final Map<UUID, Commands> commandMap;
+  private final Map<UUID, Map<SCMCommandProto.Type, Integer>> summaryMap;
```

Comment:
Would it be better if we add another class to handle Commands for one Datanode separately? Like something like DatanodeQueue. So summary and commands can go together in one place.